ports/clamav/portfile.cmake

```
@@ -3,33 +3,32 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO Cisco-Talos/clamav-devel
-  REF clamav-0.103.0
-  SHA512 e7ff4d98e0615a9fec0752bbfa2b882ae95034a8e01d0f7cc635ee520ff917c3be2a2d3273caa2fc1598e7d5ec4fd60d59b517cb439c5454d32447f8c8d7ba5a
+  REF clamav-0.104.2
+  SHA512 A2439933FC023C4263BFDAB7B79D3669BC7F38DEBA40A3E1C514E4B68F00667AF025F0D6713A4B74B7AFF58CF86AFD49305A4A91991B00431040BE5CA71FFF55
   FILE_DISAMBIGUATOR 1
   HEAD_REF master
   PATCHES
-      "build.patch"
       "cmakefiles.patch"
-      "curl.patch"
-      "mspack.patch"
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
   SOURCE_PATH ${SOURCE_PATH}
   PREFER_NINJA
   OPTIONS
```

```suggestion
vcpkg_cmake_configure(
  SOURCE_PATH ${SOURCE_PATH}
  OPTIONS
```
Please remove `PREFER_NINJA` because it is the default behaviour.